contracts/contracts/strategies/ConvexStrategy.sol

```diff
@@ -151,18 +143,29 @@ contract ConvexStrategy is BaseCurveStrategy {
     /**
      * @dev Collect accumulated CRV and CVX and send to Vault.
      */
-    function collectRewardToken() external override onlyVault nonReentrant {
+    function collectRewardTokens()
+        external
+        override
+        onlyHarvester
+        nonReentrant
+    {
         // Collect CRV and CVX
         IRewardStaking(cvxRewardStakerAddress).getReward();
-        // Send CRV
-        IERC20 crvToken = IERC20(rewardTokenAddress);
-        uint256 balance = crvToken.balanceOf(address(this));
-        emit RewardTokenCollected(vaultAddress, rewardTokenAddress, balance);
-        crvToken.safeTransfer(vaultAddress, balance);
-        // Send CVX
-        IERC20 cvxToken = IERC20(cvxRewardTokenAddress);
-        balance = cvxToken.balanceOf(address(this));
-        emit RewardTokenCollected(vaultAddress, cvxRewardTokenAddress, balance);
-        cvxToken.safeTransfer(vaultAddress, balance);
+
+        /*
+         * This code is direct copy from the same name function from InitializableAbstractStrategy
```

Comment:
Maybe we could just make the repeated section of code an internal function, and then we can call it from here.

Reply:
That makes much more sense tnx